modules/ossm-installing-gateway-using-gateway-injection.adoc

```diff
@@ -0,0 +1,257 @@
+      containers:
+        - name: istio-proxy
+          image: auto <4>
+          securityContext:
+            capabilities:
+              drop:
+              - ALL
+            allowPrivilegeEscalation: false
+            privileged: false
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          ports:
+          - containerPort: 15090
+            protocol: TCP
+            name: http-envoy-prom
+          resources:
+            limits:
+              cpu: 2000m
+              memory: 1024Mi
+            requests:
+              cpu: 100m
+              memory: 128Mi
+      securityContext:
+        sysctls:
+        - name: net.ipv4.ip_unprivileged_port_start
+          value: "0"
+      serviceAccountName: secret-reader <5>
+----
+<1> Indicates that the {istio} control plane uses the gateway injection template instead of the default sidecar template.
+<2> Ensure that a unique label is set for the gateway deployment. A unique label is required so that {istio} `Gateway` resources can select gateway workloads. 
+<3> Enables gateway injection by setting the `sidecar.istio.io/inject` label to `true`. If the name of the {istio} resource is not `default` you must use the `istio.io/rev: <istio_revision>` label instead, where the revision represents the active revision of the {istio} resource.
+<4> Sets the image field to `auto` so that the image automatically updates each time the pod starts.
+<5> Sets the `serviceAccountName` to the name of the `ServiceAccount` created previously.
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f gateway-deployment.yml
+----
+
+. Verify that the gateway `Deployment` rollout was successful by running the following command:
++
+[source,terminal]
+----
+$ oc rollout status deployment/<gateway_name> -n <gateway_namespace>
+----
++
+You should see output similar to the following:
++
+.Example output
+[source,terminal]
+----
+Waiting for deployment "<gateway_name>" rollout to finish: 0 of 1 updated replicas are available...
+deployment "<gateway_name>" successfully rolled out
+----
+
+. Create a YAML file named `gateway-service.yml` that contains the Kubernetes `Service` for the gateway. 
++
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: <gateway_name>
+  namespace: <gateway_namespace>
+spec:
+  type: ClusterIP <1>
+  selector:
+    istio: <gateway_name> <2>
+  ports:
+    - name: status-port
+      port: 15021
+      protocol: TCP
+      targetPort: 15021
+    - name: http2
+      port: 80
+      protocol: TCP
+      targetPort: 80
+    - name: https
+      port: 443
+      protocol: TCP
+      targetPort: 443
+----
+<1> When you  set `spec.type` to `ClusterIP` the gateway `Service` can only be accessed from within the cluster. If the gateway has to handle ingress traffic from outside the cluster, set `spec.type` to `LoadBalancer`. Alternatively, you can use OpenShift Routes. 
```

Comment:
The "only" should go as close as possible to the term it modifies.
```suggestion
<1> When you  set `spec.type` to `ClusterIP` the gateway `Service` object can be accessed only from within the cluster. If the gateway has to handle ingress traffic from outside the cluster, set `spec.type` to `LoadBalancer`. Alternatively, you can use OpenShift Routes. 
```

Reply:
done